### ISSUE TITLE
Ctrl + drag handle to maintain or snap the angle (as in Inkscape)

### ISSUE THREAD

**Issue by gambhiro** [mattlag/Glyphr-Studio#204]
I'm really enjoying using Glyphr!

When editing the glyph shapes I often miss the Ctrl+drag handle operation from Inkscape:

- Ctrl+drag along its angle maintains the precise angle and adjusts only the length
- Ctrl+drag away from the angle snaps in 15 degree increments

**Comment by mattlag** [comment]
Yes, I can see where this would be very useful.  Even in other programs, there is a keyboard command for drag-horizontal or drag-vertical (which would be great for whole shapes or points or handles).  The 15 degree angle snap is a good idea too.

Another alternate method that we'd probably implement is 'locking' a degree attribute of a point.  At the moment in Glyphr Studio most x/y and width/height attributes are lockable.  And we display a read-only angle attribute for points/handles.  Making this angle attribute settable and lockable is inline with your idea.

Thanks for the idea!

**Comment by pathumego** [comment]
+1
